scicomp/git-annex.rst

```diff
@@ -5,6 +5,9 @@ Git-annex for data management
 
```

Comment:
This presentation is mostly meant for people with experience with git and the command line, who want to know more about git annex. If you feel overwhelmed by the content, you may want to check the [datalad handbook](http://handbook.datalad.org/en/latest/) for a smoother entry in the use of command line and git annex functionalities.
Also note that git annex solves the issue of big files, but not numerous files (git starts to slow down with several hundreds of files). To deal with numerous files, one can use git submodules, again datalad may be a tool of choice for implementing submodules.

Reply:
```suggestion
We hope you will enjoy reading this presentation of the main functionalities of git annex. It proves interesting to follow, as to understand the logic of the tool, even if you may move to using datalad later on.
```

Reply:
I've added a lot of the above to the comment on the top (slightly rephrased but it was a good idea!)  I'm sorry for taking so long to review.